lib/nimbusRecipe.ts

```diff
@@ -41,12 +42,40 @@ export class NimbusRecipe implements NimbusRecipeType {
     const featureValue = branch.features[0].value
 
     // XXX in this case we're really passing a feature value. Hmm....
-    const template = getTemplateFromMessage(featureValue)
-    branch.template = template
-    branchInfo.template = template
-    branchInfo.surface = getDisplayNameForTemplate(template)
+    // XXX about:welcome is special, so we have to account for it
+    let template;
+    (branch.features[0].featureId === "aboutwelcome" && branch.slug != 'control')
+      ? (template = "aboutwelcome")
+      : (template = getTemplateFromMessage(featureValue));
+
+    branch.template = template;
+    branchInfo.template = template;
+    branchInfo.surface = getDisplayNameForTemplate(template);
+
+    switch (template) {
+      case "aboutwelcome":
```

Comment:
Some tests for this stanza would be good, particularly around things that seem unlikely to change much.